src/app/components/hearing/request-transcript/request-transcript-exists/request-transcript-exists.component.html

```diff
@@ -0,0 +1,19 @@
+<h1 class="govuk-heading-l">This transcript already exists</h1>
+
+<div class="govuk-button-group">
+  <a
+    id="exists-duplicate-route"
+    class="govuk-link"
+    [routerLink]="['/case', caseId, '/hearing', hearingId, '/transcripts', transcriptionId]"
```

Comment:
Remove slashes:

`[routerLink]="['/case', caseId, 'hearing', hearingId, 'transcripts', transcriptionId]"`

Prevents this:
![image](https://github.com/hmcts/darts-portal/assets/125365979/692c5874-c3be-4098-835b-743eb7882024)